__std__/nix/plutus/library/plutus-project.nix

```diff
@@ -22,6 +22,8 @@ let
 
     compiler-nix-name = library.ghc-compiler-nix-name;
 
+    evalPackages = library.pkgs;
```

Comment:
Should this not also use `library.pkgs.buildPackages.buildPackages`?

Reply:
no, in this case `library.pkgs` is the original pkgs (not from any `cross*` invocation), so the one we want.

Reply:
That doesn't make sense to me, aren't these two files literally the same thing just with one of them setup for the new infra? how are we doing cross builds in the `std` world then?

Reply:
with `plutus-project.projectCross.mingwW64`.

Reply:
More attributes over functions :( What was wrong with the old way? I won't fight about it though.

Reply:
though, using `pkgs.pkgsBuildBuild` should also work and must be the same. so we can use it instead `library.pkgs`.

Reply:
I'm very pro consistency if it doesn't cost us anything!!

Reply:
thanks. I'll close this PR then.